constructor/main.py

```diff
@@ -230,14 +228,14 @@ def main_build(
                 "Will assume it is compatible with shortcuts."
             )
         elif sys.platform != "win32" and (
-            exe_type != StandaloneExe.CONDA or (exe_version and exe_version < Version("23.11.0"))
+            exe_type != StandaloneExe.CONDA or not check_version(exe_version, min_version="23.11.0")
```

Comment:
Why are we using `not` here?

Reply:
`check_version` returns `True` when your version number is within the version specification. Using `not` here means it is `not >= 23.11.0`, which is equivalent to `< 23.11.0`.

Reply:
I think you're right, yes. That should work, too.

Reply:
I changed it - let's see what happens.